.github/workflows/authors.yml

```diff
@@ -0,0 +1,31 @@
+name: "authors update"
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: "5 0 * * 0"
+
+  workflow_dispatch:
+
+jobs:
+  authors_update:
+    if: github.repository == 'sequelize/sequelize'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: "0" # This is required to actually get all the authors
+          persist-credentials: false
+      - run: "dev/update-authors.js" # Run the AUTHORS tool
+      - uses: gr2m/create-or-update-pull-request-action@v1 # Create a PR or update the Action's existing PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          author: Node.js GitHub Bot <github-bot@iojs.org>
```

Comment:
I think we should use a different email address here. Feel free to use mail@sequelizejs.com

Reply:
Cool, Thanks!